drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -387,12 +387,7 @@ function DragAndDropTemplates(configuration) {
             popupSelector,
             {
                 style: {display: have_messages ? 'block' : 'none'},
-                attributes: {
-                    "tabindex": "-1",
-                    'aria-live': 'polite',
-                    'aria-atomic': 'true',
-                    'aria-relevant': 'all',
-                }
+                attributes: {"tabindex": "-1"}
```

Comment:
If you remove tabindex = -1, what is the impact? I'm just trying to understand why it is necessary.